contribs/gmf/examples/themeselector.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE>
+<html ng-app='app'>
+  <head>
+    <title>GeoMapFish Theme Selector Example</title>
+    <meta charset="utf-8">
+    <meta name="viewport"
+          content="initial-scale=1.0, user-scalable=no, width=device-width">
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <link rel="stylesheet" href="../../../node_modules/bootstrap/dist/css/bootstrap.css" type="text/css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+    <style>
+      ul {
+        list-style-type: none;
+      }
+    </style>
+  </head>
+  <body ng-controller="MainController as ctrl">
+    <div class="container">
+      <gmf-themeselector
+              gmf-themeselector-currenttheme="ctrl.theme">
+      </gmf-themeselector>
+      <div> The themes selected is {{ctrl.theme}}</div>
+
+    </div>
+    <p id="desc">This example shows how to use the <code>gmf.themeselector</code> directive from a c2c geoportal theme service.</p>
+
+    <script src="../../../node_modules/angular/angular.js"></script>
```

Comment:
You should add ``<script src="../../../node_modules/angular-gettext/dist/angular-gettext.js"></script>``